src/main/java/brandkon/product/ProductRestController.java

```diff
@@ -0,0 +1,48 @@
+package brandkon.product;
+
+import brandkon.product.DTO.ProductDetailed;
+import brandkon.product.DTO.ProductResponse;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class ProductRestController {
+
+    private final ProductService productService;
+
+    public ProductRestController(ProductService productService) {
+        this.productService = productService;
+    }
+
+    @GetMapping("/products")
+    public List<ProductResponse> findBrandAll(@RequestParam(required = false, value = "brandId") Long brandId) {
```

Comment:
```suggestion
    public List<ProductResponse> findBrandAll(Long brandId) {
```
참고로 이런 경우 생략했을 때랑 동작은 같습니다